hydra/core/override_parser/types.py

```diff
@@ -256,10 +257,13 @@ def value(
         else:
             return Override._convert_value(self._value)
 
-    def sweep_string_iterator(self) -> Iterator[str]:
+    def sweep_iterator(
+        self, transformer: Union[str, ElementTransformerType] = "identity"
```

Comment:
no need for the string constant. just make it lambda only. see my other comment.